### ISSUE TITLE
Errorr : 'Worker' object has no attribute '_fields'

### ISSUE THREAD

**Issue by jacky15** [mher/flower#282]
I‘m just flowing the tutorials on the celery site.
Run the command like this:
1,celery worker 
2,celery -A proj flower

but it returns an error .

Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\tornado-4.0.2-py2.7-win32.egg\tornado\web.py", line 1332, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "C:\Python27\lib\site-packages\tornado-4.0.2-py2.7-win32.egg\tornado\web.py", line 2607, in wrapper
    return method(self, *args, **kwargs)
  File "C:\Python27\lib\site-packages\flower-0.8.0-py2.7.egg\flower\views\dashboard.py", line 31, in get
    info.update(self._as_dict(worker))
  File "C:\Python27\lib\site-packages\flower-0.8.0-py2.7.egg\flower\views\dashboard.py", line 37, in _as_dict
    return dict((k, worker.__getattribute__(k)) for k in worker._fields)
  File "C:\Python27\lib\site-packages\celery-3.1.6-py2.7.egg\celery\datastructures.py", line 353, in __getattr__
    type(self).__name__, k))
AttributeError: 'Worker' object has no attribute '_fields'.

what should I do?

**Comment by asmodehn** [comment]
I have been hitting that problem recently too... celery installed by package manager on old LTS distro. flower installed by pip. versions not matching...

```
flower   -> flower:0.8.2 tornado:4.0.2 babel:1.3
software -> celery:3.1.6 (Cipater) kombu:3.0.7 py:2.7.6
            billiard:3.3.0.15 redis:2.7.2
platform -> system:Linux arch:64bit, ELF imp:CPython
loader   -> celery.loaders.app.AppLoader
settings -> transport:redis results:redis://localhost:6379
```

but not sure which version of flower is supposed to work with which version of celery...

Is there a table somewhere of versions that need to match ?
Or any chance of safe fallback being implemented sometime ?

**Comment by andresriancho** [comment]
> Is there a table somewhere of versions that need to match ?

Yup, I need that too.

Running `Celery==3.1.6` and `flower==0.8.2` yields this bug

**Comment by andresriancho** [comment]
Trying with `flower==0.7.0`

**Comment by andresriancho** [comment]
Now I get a different error with tornadoweb

> class LoginHandler(BaseHandler, tornado.auth.GoogleMixin):
AttributeError: 'module' object has no attribute 'GoogleMixin'

**Comment by andresriancho** [comment]
It's a dependency nightmare :-1:

**Comment by asmodehn** [comment]
I ended up upgrading my celery version to get it to work : 
flower   -> flower:0.8.2 tornado:4.0.2 babel:1.3
software -> celery:3.1.18 (Cipater) kombu:3.0.26 py:2.7.6
            billiard:3.3.0.20 librabbitmq:1.0.3

**Comment by gchandrasa** [comment]
@andresriancho I'm late to the party but just in case anyone stuck with the same problem

TL;DR 
If you use Celery 3.0.x, install flower with this command

```
pip install flower==0.7.3 tornado==3.2.2  
```

For more information, I wrote http://gilang.chandrasa.com/installing-flower-real-time-monitor-for-celery/

**Comment by mher** [comment]
This is already fixed. Please try the master version.
